packages/@aws-cdk/aws-lambda/lib/runtime.ts

```diff
@@ -32,6 +33,7 @@ export class Runtime {
   public static readonly DotNetCore2 =  new Runtime('dotnetcore2.0',  RuntimeFamily.DotNetCore);
   public static readonly DotNetCore21 = new Runtime('dotnetcore2.1',  RuntimeFamily.DotNetCore);
   public static readonly Go1x =         new Runtime('go1.x',          RuntimeFamily.Go);
+  public static readonly Ruby25 =       new Runtime('ruby2.5',        RuntimeFamily.Ruby);
```

Comment:
I tried to enable `inline` `{ supportsInlineCode: true }`, but it seems it isn't supported yet.

 > ZipFile can only be used when Runtime is set to either of nodejs, nodejs4.3, nodejs6.10, nodejs8.10, python2.7, python3.6.

Reply:
What do you mean, it's not supported by AWS Lambda, or you can't make it work in the CDK code?

Reply:
Hi @rix0rrr 

When I enabled `supportsInlineCode`, I got this error:

> ZipFile can only be used when Runtime is set to either of nodejs, nodejs4.3, nodejs6.10, nodejs8.10, python2.7, python3.6.

But let me try to reproduce it again and get back to you.

Reply:
Resolved by https://github.com/awslabs/aws-cdk/pull/1267#discussion_r239052825